### PR TITLE
LPS-106743 Now we're using `selectedColor` property with the current value of background color for recording the selected color

### PR DESCRIPTION
**Pull request by diegonvs**

### CODE REVIEW

**Comment by diegonvs** [comment]
ci:test:sf

File: modules/apps/analytics/.gitrepo [hunk 1]
```diff
@@ -1,8 +1,8 @@
 [subrepo]
 	autopull = false
 	cmdver = liferay
-	commit = 62e28f073f68ce63f75fe90b1aaf58781d1c30b7
+	commit = f57a144da42a8ed0dfca7f573cfe4f327103ebe8
 	mergebuttonmergecommits = false
 	mode = push
-	parent = e842984a76ad9fbf1bbecc888920ce0e1f6919ed
+	parent = 2a39eb513bef3856bcf6858b0f9fe528df2aa896
```
Comment (julien) [inline modules/apps/analytics/.gitrepo]: This change doesn't seem right @diegonvs:
Is it normal that we're updating this for this task?

Comment (julien) [reply modules/apps/analytics/.gitrepo]: Forget this comment, after rebasing with master I don't see this change.

**Comment by julien** [comment]
Submitted to https://github.com/brianchandotcom/liferay-portal/pull/83079
Thanks @diegonvs

File: modules/apps/analytics/.gitrepo (hunk 1 above)
Comment (diegonvs) [reply modules/apps/analytics/.gitrepo]: Maybe this commit was added when I rebased with brianchandotcom's  remote

**Pull request closed without merging**